### ISSUE TITLE
`id-length` should warn about the renamed identifier length in the destructuring assignment.

### ISSUE THREAD

**Issue by yeonjuan** [eslint/eslint#12827]
**Tell us about your environment**

* **ESLint Version:** v6.8.0
* **Node Version:** v12.13.0
* **npm Version:** v6.12.0

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

[Online demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQgaWQtbGVuZ3RoOiBcImVycm9yXCIqL1xuXG5jb25zdCB7IHg6IHMgfSA9IGJhcjsiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjExLCJzb3VyY2VUeXBlIjoibW9kdWxlIiwiZWNtYUZlYXR1cmVzIjp7ImpzeCI6dHJ1ZX19LCJydWxlcyI6e30sImVudiI6eyJicm93c2VyIjpmYWxzZX19fQ==)
```js
/*eslint id-length: "error"*/

const { x: s } = bar;
```

**What did you expect to happen?**

It should warn about the identifier`s` length rather than renamed `x`.

**What actually happened? Please include the actual, raw output from ESLint.**

It warns about the length of 'x';
```
3:9- Identified name 'x' is too short(<2).(id-length)
```
**Are you willing to submit a pull request to fix this bug?**

Yes.

**Comment by kaicataldo** [comment]
Agreed that this is a bug and should be fixed.
